### PR TITLE
Upgrade from 4.5.latest to 4.6.0 without Headless

### PR DESCRIPTION
**Pull request by adriendupuis**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | 
| Versions      | 
| Edition       | 

Fork of #2276 to temporally remove the not-fixed-yet ibexa/content 4.5.5 to ibexa/headless 4.6.0 upgrade.

It allow to offers upgrade migration steps to Experience and Commerce user in the meantime.

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/update_and_migration/from_4.5/update_from_4.5.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+php bin/console ibexa:migrations:import vendor/ibexa/installer/src/bundle/Resources/install/migrations/2023_12_07_20_23_editor_content_type.yaml --name=2023_12_07_20_23_editor_content_type.yaml
+php bin/console ibexa:migrations:import vendor/ibexa/installer/src/bundle/Resources/install/migrations/2024_01_09_22_23_editor_permissions.yaml --name=2024_01_09_22_23_editor_permissions.yaml
+php bin/console ibexa:migrations:migrate --file=2023_12_07_20_23_editor_content_type.yaml --file=2024_01_09_22_23_editor_permissions.yaml
+```
+
+More information ...
+
+### Site Context
+
+Site context is ...
```
Comment (dabrt) [inline docs/update_and_migration/from_4.5/update_from_4.5.md:168]:
```suggestion
### Site context

Site context is used in Content Tree to display only those Content items that belong to the selected website.
```

File: docs/update_and_migration/from_4.5/update_from_4.5.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+Add non publicly accessible locations to the list of excluded paths:
```
Comment (dabrt) [inline docs/update_and_migration/from_4.5/update_from_4.5.md:170]:
```suggestion
You can add locations that shoudn't be publicly accessible to the list of excluded paths:
```

File: docs/update_and_migration/from_4.5/update_from_4.5.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+You can use your own content type representing backoffice user or use default one provided by Ibexa DXP:
+
+```bash
+php bin/console ibexa:migrations:import vendor/ibexa/installer/src/bundle/Resources/install/migrations/2023_12_07_20_23_editor_content_type.yaml --name=2023_12_07_20_23_editor_content_type.yaml
+php bin/console ibexa:migrations:import vendor/ibexa/installer/src/bundle/Resources/install/migrations/2024_01_09_22_23_editor_permissions.yaml --name=2024_01_09_22_23_editor_permissions.yaml
+php bin/console ibexa:migrations:migrate --file=2023_12_07_20_23_editor_content_type.yaml --file=2024_01_09_22_23_editor_permissions.yaml
+```
+
+More information ...
```
Comment (dabrt) [inline docs/update_and_migration/from_4.5/update_from_4.5.md:164]:
```suggestion
More information ...
```
Do you remember what you wanted to say here?

File: docs/update_and_migration/from_4.5/update_from_4.5.md [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+                    - /1/61/    # Product categorises
+                    - /1/65/    # Corporate Account
+                    - /1/57/    # Tags
+```
+
+## Revisit optional configuration
+
+### Activity Log
+
+By default, activity log keeps entries for 30 days. You can change this value by setting `ibexa.repositories.<name>.activity_log.truncate_after_days` parameter:
```
Comment (dabrt) [inline docs/update_and_migration/from_4.5/update_from_4.5.md:194]:
```suggestion
By default, activity log keeps entries for 30 days. 
You can change this value by setting `ibexa.repositories.<name>.activity_log.truncate_after_days` parameter:
```

File: docs/update_and_migration/from_4.5/update_from_4.5.md [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+            # ...
+            activity_log:
+                truncate_after_days: 10
+```
+
+## Revisit permissions
+
+### Recent activity
+
+"Activity Log / Read" policy (`activity_log/read`) should be added to every role having access to the Back Office, at least with the "Only own log" limitation.
```
Comment (dabrt) [inline docs/update_and_migration/from_4.5/update_from_4.5.md:209]:
```suggestion
You must add the "Activity Log / Read" policy (`activity_log/read`) to every role that has access to the Back Office, at least with the "Only own log" limitation.
```

File: docs/update_and_migration/from_4.5/update_from_4.5.md [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+This policy is mandatory to display the "Recent activity" block in [dashboards](#dashboard), and the "Recent activity" block in [user profiles](#user-profile). 
+
+The following migration example allows `Editor` role to access own activity log:
```
Comment (dabrt) [inline docs/update_and_migration/from_4.5/update_from_4.5.md:212]:
```suggestion
The following migration example allows users with the `Editor` role to access their own activity log:
```

**Review by dabrt — CHANGES_REQUESTED** [review]

File: docs/update_and_migration/from_4.5/update_from_4.5.md (hunk 3 above)
Comment (adriendupuis) [reply docs/update_and_migration/from_4.5/update_from_4.5.md:164]: @adamwojs Do you remember what kind of extended information you were thinking of?

Comment (adriendupuis) [reply docs/update_and_migration/from_4.5/update_from_4.5.md:164]: Removed

**Comment by adriendupuis** [comment]
There is few issues on other editions' upgrade. To exclude Headless is not enough. Efforts are concentred on #2276

**Pull request closed without merging**